src/blocks/kdeconnect.rs

```diff
@@ -129,6 +137,22 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
                     }
                 }
 
+                let network = device.network().await;
+                if let Some(net) = network.0 {
+                    let network_strength = network.1.unwrap_or(0);
+                    values.insert(
+                        "network_icon".into(),
+                        Value::icon(api.get_icon("net_wireless")?),
```

Comment:
It would be nice (but not a must) if this were an icon progression.

There's some code in this PR: https://github.com/greshake/i3status-rust/pull/1728/files#diff-f32290632a1d310d9d47a251e3ce1ed66f1a6943421ee2e44da0c8bfb5e118caR277-R286

that can be used for an icon progression if the scale isn't 0%-100% (0f64..=1f64)

Reply:
Oh Yeah, the Icon Progression is something I tried first and got really frustrated because it didn't work. Turns out the Font Awesome 6's network related progress icons were for paid version only, and my arch install didn't have that. I haven't looked at other icon themes yet. If we can make it work for some, then I'll modify it to be that case. I looked at the network block and in fontawesome6 it was not a progression.

I'll try to use nordic font(?) and see if I can make it work.

Reply:
material-nf is the only font with the net_wireless progression.

Reply:
Should be fixed now. I didn't test using the nordic-nf myself though.

Reply:
They aren't nordic-nf, but material-nf :) This change looks good though!